### PR TITLE
feat: update BEP153Height of testnet

### PR DESCRIPTION
**Pull request by Loverush**
This PR aims to enable BEP153 on testnet.

The BNB Beacon Chain testnet is expected to have a scheduled hard fork upgrade at block height 30,689,000. Based on the current block generation speed, the hard fork is forecasted to occur on 16 Sep. at 7:30 (UTC). The full node runners on testnet must switch their software version to [v0.10.2](https://github.com/bnb-chain/node/releases/tag/v0.10.0) by 16 Sep..

### CODE REVIEW

**Review by guagualvcha — APPROVED** [review]

**Review by guagualvcha — CHANGES_REQUESTED** [review]
checking

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # Changelog
 
-## 0.10.1
+## 0.10.2
 FEATURES
 * [\#878](https://github.com/bnb-chain/node/pull/878) [CI] Add ci script to docker image
 
 IMPROVEMENTS
```
Comment (forcodedancing) [inline CHANGELOG.md:7]: It seems it should be `FEATURES`

File: asset/testnet/app.toml [hunk 2]
```diff
@@ -39,6 +39,7 @@ FixZeroBalanceHeight = 1
 BEP70Height = 1
 BEP67Height = 1
 BEP8Height = 1
+BEP153Height = 30516226
```
Comment (forcodedancing) [inline asset/testnet/app.toml:42]: Can we move it below `BEP151Height = 28250000` ?

File: CHANGELOG.md (hunk 1 above)
Comment (guagualvcha) [reply CHANGELOG.md:7]: Agree

File: asset/testnet/app.toml (hunk 2 above)
Comment (guagualvcha) [reply asset/testnet/app.toml:42]: Nice catch

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
 # Changelog
 
-## 0.10.1
+## 0.10.2
 FEATURES
 * [\#878](https://github.com/bnb-chain/node/pull/878) [CI] Add ci script to docker image
```
Comment (forcodedancing) [inline CHANGELOG.md:5]: Add ci scripts to `build` docker image

**Review by owen-reorg — APPROVED** [review]

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**